spring-cloud-dataflow-docs/src/main/asciidoc/streams-developer-guide.adoc

```diff
@@ -98,15 +98,16 @@ To do so:
 
 . Visit the link:https://start.spring.io/[Spring Initialzr] site.
 .. Create a new Maven project with a *Group* name of `io.spring.stream.sample` and an *Artifact* name of `transformer`.
-.. In the Dependencies text box, type `stream` to select the Spring Cloud Stream dependency.
-.. In the Dependencies text box, type either `kafka` or `rabbit` and select which middleware you want to use.
+.. In the Dependencies text box, type `stream` to select the `Cloud Stream` dependency.
+.. In the Dependencies text box, type `web` to select the `Web` dependency.
+.. In the Dependencies text box, select the middleware required by typing either: `kafka` and select the `Kafka Streams` dependency or `rabbit` and select the `RabbitMQ` dependency.
```

Comment:
If the example is supposed to work with Kafka, we should select `Cloud Stream` and `Kafka`. The `Kafka Streams` option is specifically for Kafka Streams binder implementation. Likewise, to make it work with RabbitMQ, the dependencies should be `Cloud Stream` and `RabbitMQ`.

Reply:
Updated the Document PR to replace `Kafka Streams` with `Kafka`.